app/services/mailers/launch_reminder_comms.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
+
+class Mailers::LaunchReminderComms
```

Comment:
Maybe `class Comms::LaunchReminder`?

Reply:
Thanks for the suggestion, I decided to use `BulkMailers` for the module name 👍